README.md

```diff
@@ -1,2 +1,187 @@
 # ea_enterprise
-Collection of containers to run Enroll in development context
+Collection of containers to run Enroll in a development context
+
+## Getting Started
+
+Download and install Docker Desktop.
+
+Clone the ea_enterprise repo:
+```
+git clone https://github.com/ideacrew/ea_enterprise.git
+```
+
+Copy the .env.example to .env and change the values as needed (don't commit .env file)
+```
+cp .env.example .env
+```
+
+Ensure the other repositories are present on your local machine and at the same level as ea_enterprise:
+
+- enroll
+- fdsh_gateway
+- medicaid_gateway
+- medicaid_eligibility
+- polypress
+
+the layout should look like this (you can change the *projects* folder)
+```
+~ /
+└── projects/
+    ├── ea_enterprise
+    ├── enroll
+    ├── fdsh_gateway
+    ├── medicaid_gateway
+    └── polypress
+
+```
+
+Be sure all repositories are up to date with their respective release branches before running any commands!
+
+## important concepts
+
+- inside the container:
+
+- attach to a container:
+
+- outside the container:
+
+## Known workarounds
+
+The services `fdsh_gateway`, `enroll` has "localhost" as the "server" on config/mongoid.yml, the docker-compose configuration "patches" this by mounting anther configuration, the "patched" configuration is located at `config/mongoid.yml.docker` and is mounted on the container at `/{APP}/config/mongoid.yml`
+
+Local aca_entites: it is already mounted and can be changed on the gemfile, the trick is just to restart enroll;
```

Comment:
We should add some working that aca_entities should be in the projects folder as well for this. seems obvious but someone might miss

Reply:
added on the previous section about checking out repos

Reply:
This also obvious, but may be helpful to word this more generally.  Something like:

"Local aca_entites: it is already mounted and can be changed on the Gemfile, the trick is just to restart the service you are working with (i.e., restart only Enroll if you are updating Enroll's Gemfile)."

Reply:
fixed!